utils/parser.py

```diff
@@ -51,8 +55,10 @@ def get_message_text(message_struct: dict):
             message_text = message_text[0]
         else:
             message_text = ""
-
-        return message_text.lower().strip()
+        try:
+            return message_text.lower().strip()
+        except Exception as err:
+            logging.exception(f'\nError occurred with argument: {message_text}')
```

Comment:
Very good Nurik 👏 
Great use of **f** strings! 

Couple of notes:
1. It is generally better to pass variables as arguments to loggers. Don't ask why, it's just the way it's implemented and it's a pattern for python developers. Also it's way faster:
<img width="677" alt="Screenshot 2020-08-11 at 20 47 27" src="https://user-images.githubusercontent.com/4043765/89936566-2798bf80-dc14-11ea-9ff8-ee02b5376f8c.png">

So I urge to rewrite the logger to NOT use f strings

2. Also could you pass the full `message_struct` rather than `message_text` please?

3. Also you logged but didn't raise the exception. So at the end the code should fail with an error, it's correct behaviour.
So just add `raise` at the end

Reply:
1. Used format string instad cause familiar with it
2. done
3. added just raise. However couldnt insert message to exception. and what is the difference btwn 'raise' and 'raise err'  
`        except Exception as err:
            logging.exception('\nError occurred with argument: {}'.format(message_struct))
            raise  # why this dont work = raise err('Please check argument {}'.format(message_struct))`

Reply:
should I do pull request again?

Reply:
Nono, keep this pr. Just make changes and commit them to the same branch. The PR will then be updated automatically

Reply:
>Used format string instad cause familiar with it

I know I'm just giving you valuable hints :)

Reply:
For point 3, 
`err` is an instance already, you cannot initiate it again. If I understood you correctly you want to raise the previous exception with extra params. But it's unnecessary.
You could do just raise and it would be okay since we are already recording the full error  trace for debugging in a file.